sops/node-specific/GDI-SOP0008_node-helpdesk-ticket-classification.md

```diff
@@ -0,0 +1,487 @@
+
+  - Password issue - user having an issue with their password
+
+  - User account
+
+- **Distribution**
+
+  - File issues - to be used for “truncated files” which could have been submitted truncated or are truncated due to error in download. Also missing files from a dataset
+
+  - Metadata issues - users asking further info on metadata e.g., trying to understand how it links, asking if more data available
+
+  - Metadata query - user asking how to download metadata or asking for AF metadata
+
+    - **Distribution\_Download\_Account**
+
+       - Datasets/files request not in client -  used for files not available with the download tool
+
+       - Alternative Download - to be used when a user cannot download using any of our download tools
+
+    - **Distribution Keys**
+
+       - Key request
+
+    - **Distribution\_Download\_Client**
+
+       - API Download - to be used 
+
+       - Dataset/file requesting issues
+
+       - Decryption/unzipping problems
+
+       - Download streamer
+
+       - Download streamer 500
+
+       - Download streamer issues
+
+       - Firewall issues
+
+       - Log in problems
+
+       - Null/Sleeping timeout errors
+
+       - Download client issues
+
+       - User error
+    
+       - Version issue
+
+- **DAC** 
+
+  - DAC tool query - to be used when DAC is having issues with tools 
+
+  - DAC curation - to  be used when we do DAC curation, and update DACs for DACs i.e., change DAC members
+
+  - Update DAC - use for contact and detail changes
+
+  - DAC referred - to be used when we have to point a user back to a DAC e.g., they say they have been granted access but we can’t see that they have
+
+  - DAC request to create account - DAC requesting to creating a user account
+
+* **Archiving**
+
+  - Data deposition - Archiving issue
+
+  - Archiving state - file is still processing somewhere
+
+  - Failed state - file has failed to archive
+
+  - Tags for different files status 
+
+- **External Comms**
+
+  - Journal communications - to be used when in communication with Journals
+
+* **Security**
+
+  - EGA security issues - to be used if there is a security breach
+
+
+- **Delete**
+
+  - Non - issue - to be used when user has submitted a ticket with an issue, and then replied to the ticket saying “all ok after all” before we have had chance to reply
+
+  - Spam
+
+* **Consortium**
+
+  - Tags for different consortiums
+ 
+
+* #### Ticket priority
+
+We propose the use of three priority levels as listed below. However, each node might decide to use other levels.
+
+- **Low** - suitable for tickets that are expected to be resolved quickly and which are of routine character
+
+- **Medium** - suitable for tickets that are more complicated and might take longer time to resolve
+
+- **High** - suitable for tickets that need to be handled immediately, e.g. security breaches
+
+* #### Ticket assignee
+
+Name of node helpdesk officer assigned to the ticket.
```

Comment:
I'm wondering if the name of one person is enough, or if we need to add:
- The email of that person (?)
- More people that are to be kept in the loop for the issue (?)